platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/camera/CameraAnimatorActivity.java

```diff
@@ -116,20 +182,66 @@ public void onAnimationUpdate(ValueAnimator animation) {
     return tiltAnimator;
   }
 
-  private static class LatLngEvaluator implements TypeEvaluator<LatLng> {
+  //
+  // Interpolator examples
+  //
 
-    private final LatLng latLng = new LatLng();
+  private Animator createExampleInterpolator(int menuItemId) {
```

Comment:
One minor thing, what about changing the name of the method here to `obtainExampleAnimator(int menuItemId)` instead? We're not actually creating the animators here and there's another method with the same exact name (`createExampleInterpolator(Interpolator interpolator, long duration)`) which makes it kinda confusing.